symfony.lock

```diff
@@ -50,6 +50,9 @@
     "doctrine/dbal": {
         "version": "2.10.2"
     },
+    "doctrine/deprecations": {
```

Comment:
The change to the symfony.lock seems unrelated to this PR. Was there a good reason to add it? Otherwise please remove it from history ;)